.travis.yml

```diff
@@ -11,9 +11,15 @@ install:
 services: 
   - postgresql
 
+addons:
+  postgresql: '9.4'
+  apt:
+    packages:
+      - postgresql-server-dev-9.4
+
```

Comment:
What does this do?

Reply:
It updates the postgress database to be used in travis